### DISCUSSION TITLE
Is there more information about the instruction `mma.sync.aligned.shape.row.col.s32.b1.b1.s32.bitOp.popc` for single bit mma?

### DISCUSSION THREAD

**Discussion by luliyucoordinate** [NVIDIA/cutlass#1761] [Q&A] [answered]
I noticed that the documentation mentioned, "For single-bit mma.sync, multiplication is replaced by a sequence of logical operations; specifically, mma.xor.popcand mma.and.popc computes the XOR, AND respectively of a k-bit row of A with a k-bit column of B, then counts the number of set bits in the result (popc)." Is there any related documentation about how multiplication is achieved through a sequence of logical operations?

**Comment by thakkarV** [comment, marked as answer]
multiply and adds are defined as logical and and logical or operations for boolean operands https://en.wikipedia.org/wiki/GF(2)
